src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -286,4 +275,83 @@ void getHearingChannels_ifAdjourmentEnabled_getNextHearing() {
             .containsOnly(TELEPHONE);
     }
 
+    @DisplayName("When adjournment flag is false, returns hearing chanel from the case")
+    @Test
+    void getHearingChannels_ifAdjournmentEnabledAndNextHearingIsNull_returnDefaultHearingChannel() {
```

Comment:
I think this test can be deleted? The name doesn't match the contents, and all the scenarios I believe are covered in the tests below. Double check, but consider removing.

Reply:
Removed the unwanted test case